eliot-service/eliot/liblogging.py

```diff
@@ -109,4 +109,4 @@ def log_config(logger, config_manager, component):
             value = "*****"
 
         full_key = generate_uppercase_key(key, ns).upper()
-        logger.info(f"{full_key}={value}")
+        logger.debug(f"{full_key}={value}")
```

Comment:
This changes the settings logging to be at the DEBUG level. We don't need to log the settings at INFO level every time things reboot.

Reply:
the way this is phrased implies DEBUG has less information than INFO, but if I'm not mistaken DEBUG contains INFO, so I'm confused.

Reply:
I don't understand what you mean by that.

With Python logging, you can log at different levels. Then the application sets the level below which logging calls are ignored. Our server set up is at INFO, but local dev environment is set up at DEBUG. So by changing this from INFO to DEBUG, we will continue to see these lines (which spit out all the configuration) in the logs in a local dev environment, but not in a server environment.

Reply:
yes it does. My bad, it was my misinterpretation.